src/geventhttpclient/utils.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
these functions probably dont have much use outside of useragent, so maybe they could be private class methods there instead?

Reply:
Done!